src/main/java/com/celements/web/classes/oldcore/XWikiRightsClass.java

```diff
@@ -0,0 +1,69 @@
+package com.celements.web.classes.oldcore;
+
+import java.util.List;
+
+import javax.inject.Singleton;
+
+import org.xwiki.component.annotation.Component;
+
+import com.celements.model.classes.AbstractClassDefinition;
+import com.celements.model.classes.fields.BooleanField;
+import com.celements.model.classes.fields.ClassField;
+import com.celements.model.classes.fields.list.AccessRightLevelsField;
+import com.celements.model.classes.fields.list.ListOfGroupsField;
+import com.celements.model.classes.fields.list.ListOfUsersField;
+import com.celements.rights.access.EAccessLevel;
+import com.xpn.xwiki.user.api.XWikiUser;
+
+@Singleton
+@Component(XWikiRightsClass.CLASS_DEF_HINT)
+public class XWikiRightsClass extends AbstractClassDefinition implements IOldCoreClassDef {
+
+  public static final String CLASS_SPACE = "XWiki";
+  public static final String CLASS_NAME = "XWikiRights";
+  public static final String CLASS_FN = CLASS_SPACE + "." + CLASS_NAME;
+  public static final String CLASS_DEF_HINT = CLASS_FN;
+
+  public static final String FIELD_NAME_ALLOW = "allow";
+  public static final String FIELD_PRETTY_NAME_ALLOW = "Allow/Deny";
+  public static final String FIELD_NAME_ACCESSLVL = "levels";
+  public static final String FIELD_PRETTY_NAME_ACCESSLVL = "Levels";
+  public static final String FIELD_NAME_USERS = "users";
+  public static final String FIELD_PRETTY_NAME_USERS = "Users";
+  public static final String FIELD_NAME_GROUPS = "groups";
+  public static final String FIELD_PRETTY_NAME_GROUPS = "Groups";
```

Comment:
using Strings for fields should be discouraged, better remove them. if really needed one could always use FIELD_X.getName()